flybywiresim/a32nx
# fix: show button labels when unlit

## Summary of Changes

- Fixed button labels not showing when unlit
- Fixed white dots illuminating when ANN LT TEST is active
## Screenshots (if necessary)


Before:
![image](https://user-images.githubusercontent.com/16512508/105651705-70fc6480-5e6c-11eb-834f-d404bc388707.png)
![image](https://user-images.githubusercontent.com/16512508/105651858-d2243800-5e6c-11eb-923d-ac8dd1ca0d8e.png)
After:
(it's very subtle so you will need to zoom in to see)
![image](https://user-images.githubusercontent.com/16512508/105651735-81acda80-5e6c-11eb-9d12-ad3218167193.png)
![image](https://user-images.githubusercontent.com/16512508/105651946-0c8dd500-5e6d-11eb-8370-d37ccafb4aae.png)


## References


![image](https://upload.wikimedia.org/wikipedia/commons/4/43/Overhead_panel_of_an_Airbus_A320_during_cruise.jpg)
![image](https://user-images.githubusercontent.com/39596827/96653688-ef131180-1339-11eb-8925-831514423c09.JPG)




## Additional context





## Testing instructions

- Check that all buttons have visible labels when unlit
- Ensure that all buttons still work normally when pressed and when ANN LT TEST is active


## How to download the PR for QA

Every new commit to this PR will cause a new A32NX artifact to be created and uploaded.
The build script will have already been run with the latest changes, so no need to rerun it once you download the zip.

1. Make sure you are signed in to GitHub
1. Click on the **Checks** tab on the PR
1. On the right side, click on the **Artifacts** drop down and click the **A32NX** link
